pyproject.toml

```diff
@@ -3,6 +3,127 @@ name = "openpilot"
 requires-python = ">= 3.11"
 readme = "README.md"
 license = {text = "MIT License"}
+version = "0.1.0"
+description = "an open source driver assistance system"
+authors = ["Vehicle Researcher <user@comma.ai>"]
+dependencies = [
+
+# multiple users
+"sounddevice >= 0",  # micd + soundd
```

Comment:
what's the >= 0 for?

Reply:
Since we are using the --preview mode (in order to use github packages), we have to specify at least a lower bound otherwise uv will print a warning. This is equivalent to sounddevice="*"